recipe/meta.yaml

```diff
@@ -11,30 +11,34 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python >=3.7
     - pip
     - pytest-runner
+    - setuptools_scm
   run:
     - python >=3.7
     - xarray
     - pandas
     - numpy
+    - importlib-metadata
 
 test:
   imports:
     - cmip6_preprocessing
+    
+  commands: python -c "import cmip6_preprocessing; assert cmip6_preprocessing.__version__ == '{{ version }}', cmip6_preprocessing.__version__"
 
 about:
   home: https://github.com/jbusecke/cmip6_preprocessing
   license: MIT
   license_family: MIT
   license_file: LICENSE.txt
-  summary: Some code to homogenize naming and parse variables in a xgcm compatible xarray dataset
+  summary: Some code to homogenize naming and parse variables in an xgcm compatible xarray dataset
```

Comment:
We might want to update this a bit, but I have some cleaning/refactoring/renaming to do and will do it then I think